vpr/src/pack/sync_netlists_to_routing_flat.cpp

```diff
@@ -378,6 +359,15 @@ static void sync_clustered_netlist_to_routing(void) {
             }
             PinType pin_type = node_type == OPIN ? PinType::DRIVER : PinType::SINK;
 
+            /* Pin already exists. This means a global was connected to here. */
```

Comment:
I think you should say it was an "a global net that was not routed (i.e. 'ideal' mode)"